functions/__fzf_preview_file.fish

```diff
@@ -3,8 +3,13 @@ function __fzf_preview_file --argument-names file_path --description "Print a pr
     if test -f "$file_path" # regular file
         bat --style=numbers --color=always "$file_path"
     else if test -d "$file_path" # directory
-        set --local CLICOLOR_FORCE true
```

Comment:
This wasn't working and I couldn't figure out a way to pass in `CLICOLOR_FORCE` while still using `command` to execute ls (which is necessary when people have ls aliased to something else), so I am removing it. Hopefully, passing in the additional `-F` will compensate a bit for the lack of colors.